lib/commands/create-migration.js

```diff
@@ -61,6 +61,7 @@ async function executeCreateMigration (internals, config) {
       path += folder[i] + '/';
     }
   }
+  internals.argv['migrations-dir'] = path
```

Comment:
Why do you overwrite the initial state here? That is very dangerous and I see it breaking multi scope execution.

Reply:
createSqlFiles uses the internals.argv['migrations-dir'] for the target directory. To avoid updating internals, we'll need to setup another mechanism to pass down the target directory.

There's precedent of internals being updated:
`  internals.argv.title = folder[folder.length - 1] || folder[0];`

Reply:
I'll add a test.